src/t8_schemes/t8_standalone/t8_standalone_implementation.hxx

```diff
@@ -1584,10 +1593,9 @@ struct t8_standalone_scheme
    * Note Caller is responsible for taking the absolute value of leveldiff
   */
   static constexpr t8_linearidx_t
-  num_descendants_at_leveldiff (const t8_element_t *elem, const t8_element_level leveldiff) noexcept
+  num_descendants_at_leveldiff ([[maybe_unused]] const t8_element_t *elem, const t8_element_level leveldiff) noexcept
   {
-    if (leveldiff < 0)
-      return 0;
+    T8_ASSERT (leveldiff <= get_maxlevel ());
```

Comment:
@Davknapp I fixed #1436 here as well, but didn't realize when we talked.

leveldiff is actually unsigned, so this branch could never be entered and Wextra warned me about it